README.md

```diff
@@ -789,8 +789,8 @@ If you'd like to take part in the development processes, here are a few things t
 
 * Summoner is a multi-package project which has two packages inside: `summoner-cli` and `summoner-tui`. We are supporting the same functionality in both of them, so, if it's possible, you should add/change the code in both of the packages.
 * Summoner uses the custom prelude [`relude`](https://github.com/kowainik/relude).
-* We are using the [Kowainik style guide](https://github.com/kowainik/org/blob/master/style-guide.md#haskell-style-guide).
-* To sum up, [here is the Contributing guide](https://github.com/kowainik/.github/blob/master/CONTRIBUTING.md#contributing-to-the-kowainik-repositories) we use across the repositories.
+* We are using the [Kowainik style guide](https://github.com/kowainik/org/blob/main/style-guide.md#haskell-style-guide).
```

Comment:
I just noticed, that this could be a link to our blog post, not the markdown file 🙂